docs/pages/components/button.md

```diff
@@ -173,28 +177,135 @@ Button with multiple actions
 {% endcapture %}
 {% include display-component.html component=button-split %}
 
-## Button RTL support
+<br>
+
+## Content Density - Button Sizes
+
+All the buttons support the cozy and compact form factors.
+
+{% capture button %}
+<button class="fd-button">Save</button>
+<button class="fd-button fd-button--standard sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--ghost sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--emphasized sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--transparent sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--positive sap-icon--accept">Approve</button>
+<button class="fd-button fd-button--negative sap-icon--decline">Reject</button>
+<button class="fd-button fd-button--attention sap-icon--warning">Attention</button>
+<br><br>
+<button class="fd-button fd-button--compact">Edit</button>
+<button class="fd-button fd-button--standard fd-button--compact">Edit</button>
+<button class="fd-button fd-button--ghost fd-button--compact">Edit</button>
+<button class="fd-button fd-button--emphasized fd-button--compact sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--transparent fd-button--compact sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--positive fd-button--compact sap-icon--accept">Approve</button>
+<button class="fd-button fd-button--negative fd-button--compact sap-icon--decline">Reject</button>
+<button class="fd-button fd-button--attention fd-button--compact sap-icon--warning">Attention</button>
+{% endcapture %}
+{% include display-component.html component=button %}
+
+<br>
+
+## Icon and Text
+
+The buttons can have icon and/or text. The recommendation is to have either text or icon. Use icon for buttons only for very basic standard icon metaphores (e.g. _Back to previous screen, Create a new item, Remove from list, Edit, ..._)
```

Comment:
The buttons can contain icons and/or text. The recommendation is to use either one or the other, not both. Use icon for buttons that contain very basic standard icon metaphors (e.g. _Back to previous screen, Create a new item, Remove from list, Edit, ..._).